src/app.js

```diff
@@ -0,0 +1,53 @@
+let list = []
+// const tableBody = document.querySelector("tableBody")
+
+
+const startApp = async () => {
+  const result = await fetchList()
+  if (result) {
+     list = result
+    populateTable()
+  }
+}
+
+const fetchList = async (page = 1) => {
+  const list = await fetch(
+    "https://randomapi.com/api/8csrgnjw?key=LEIX-GF3O-AG7I-6J84" +
+      `&page=${page}`
+  ).then((result) => result.json())
+  return list
+}
+
+function populateTable() {
+  const table = document.getElementById("tableBody")
+    const male = list.results[0][2]
```

Comment:
Codacy has a fix for the issue: [Replace `····const·male·=·list.results[0][2]` with `··const·male·=·list.results[0][2];`](https://app.codacy.com/gh/ZeeUnltd/starter/pullRequest?prid=10061291)
```suggestion
  const male = list.results[0][2];
```